Zapper-fi/studio
# feat(coslend): add coslend app integration

## Description

add [coslend](https://coslend.com) app integration

- supply (token) fetcher and cache manager
- borrow contract position fetcher
- balance fetcher
- TVL fetcher

## Checklist

- [x] (optional) As a contributor, my Twitter handle is: https://twitter.com/coslend

## How to test?

1. edit/create .env adding ENABLED_APPS="coslend"
2. run pnpm dev to spin up nest server
3. visit dev endpoints below to see result of fetchers to integrate coslend:

- http://localhost:5001/apps/coslend/tokens?network=evmos&groupIds[]=supply
- http://localhost:5001/apps/coslend/positions?network=evmos&groupIds[]=borrow
- http://localhost:5001/apps/coslend/tvl?network=evmos
- http://localhost:5001/apps/coslend/balances?network=evmos&addresses[]=<wallet_address>
